api/src/services/application-csv-export.service.ts

```diff
@@ -639,6 +656,83 @@ export class ApplicationCsvExporterService
           });
       });
 
+    // add programs to csv headers
```

Comment:
A lot of this looks like it might be copy-pasta from above and just changing the `preferences` --> `programs`? Could be nice to share this piece

Reply:
you right, updated!